docs/dev-guide/installingNative.md

```diff
@@ -93,11 +95,22 @@ Most of these prerequisites can be installed using the package manager of your O
     The main prerequisites can be installed with [Homebrew](http://brew.sh/):
 
     ```sh
-    brew install git graphviz postgresql redis uv d2 node npm pgvector
+    brew install git graphviz postgresql@17 redis uv d2 node pgvector
```

Comment:
Don't forget to fix the Postgres version per my other comments.

Reply:
postgres 18 is now out as well, it may be time for an upgrade soon.

Reply:
CHanged in 94d74ac4336856b928672dc27b716b20d4f1ba4a.

Reply:
We're tied to what RDS supports for Aurora Serverless v2, which is currently 16.